fvm/state/state.go

```diff
@@ -53,6 +55,15 @@ func NewState(ledger Ledger, opts ...StateOption) *State {
 	return ctx
 }
 
+func (s *State) Child() *State {
+	cs := defaultState(s)
+	cs.interactionUsed = s.interactionUsed
+	cs.maxKeySizeAllowed = s.maxKeySizeAllowed
+	cs.maxValueSizeAllowed = s.maxValueSizeAllowed
+	cs.maxInteractionAllowed = s.maxInteractionAllowed
+	return cs
+}
```

Comment:
where do we merge the child to the parent?

Reply:
you mean this line https://github.com/onflow/flow-go/pull/208/files/f49fff223aa1269b3f0b22ffc435d58850038222#diff-4e9aadc99d611cd3a369ca71b29ef7fc9dd2beb4f88169978c3a32c3bcbed2f5R206 ?